docker/statutory/Dockerfile

```diff
@@ -20,9 +22,8 @@ USER node
 ENV NPM_CONFIG_PREFIX=/home/node/.npm-global
 ENV PATH="/home/node/.npm-global/bin:${PATH}"
 
-RUN npm install -g --loglevel warn nodemon bunyan && npm cache clean --force
-RUN npm install --loglevel warn
+RUN npm install
```

Comment:
why removing global packages installation though? seems like you won't be able to run `nodemon` and `bunyan` directly, that's why it was there

Reply:
Ah, so we're installing them as global so we can use them directly. That makes sense, I will re-add them

Reply:
Either this (which seems more fancy to me) or it can also be done in 2 other ways:
- calling nodemon as ./node_modules/.bin/nodemon (and same with bunyan)
- calling nodemon as npx nodemon (same with bunyan)

Reply:
I've put it back